packages/perseus/src/widgets/interactive-graphs/backgrounds/axis-labels.tsx

```diff
@@ -75,7 +75,22 @@ export default function AxisLabels({i18n}: {i18n: I18nContextType}) {
         </>
     );
 }
-
+// This function clamps the label position to ensure that the labels do not go too far
+// outside the graph bounds. This is only required when the labelLocations are set to "onAxis".
+export const clampLabelPosition = (
+    labelPosition: vec.Vector2,
+    graphInfo: GraphDimensions,
+): vec.Vector2 => {
+    const x = Math.max(
+        Math.min(labelPosition[X], graphInfo.width + fontSize * 1.25), // The maximum x value is the width of the graph + 2 font sizes
```

Comment:
```suggestion
        Math.min(labelPosition[X], graphInfo.width + fontSize * 1.25), // The maximum x value is the width of the graph + 1.5 font sizes
```

Reply:
Super duper teeny tiny nit: Can you put this longer comment on the line above the code? It seems to be going kind of far on the side to me.